docs/source/recipes.rst

```diff
@@ -269,6 +270,8 @@ You can also provide an optional ``increment_by`` argument which will modify inc
     >>> 5.75
     customer.appointment
     >>> datetime.date(2014, 7, 22)
+    customer.name
+    >>> 'Custom num: 5'
```

Comment:
Shouldn't the code be placed after `>>>`? :sweat_smile:

```
 >>> customer.name
'Custom num: 5'
```

Reply:
Fixed by 786226d